testing/functional_placeholders_arithmetic.cu

```diff
@@ -66,7 +66,7 @@ template<typename T>
 {
   __host__ __device__ T operator()(const T &x) const
   {
-    return +x;
+    return static_cast<T>(+x);
```

Comment:
This is here because `+x` may cause promotion - if `T` is `char`, `+x` would be promoted to `int`. Can we leave a comment explaining this?

Reply:
Done.